src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -53,15 +55,24 @@ class TestAttrAccess : public AttributeAccessInterface
     TestAttrAccess() : AttributeAccessInterface(Optional<EndpointId>::Missing(), TestCluster::Id) {}
 
     CHIP_ERROR Read(const ConcreteAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+    CHIP_ERROR Write(const ConcreteAttributePath & aPath, TLV::TLVReader & aReader, bool * aDataWrite) override;
 
 private:
     CHIP_ERROR ReadListInt8uAttribute(AttributeValueEncoder & aEncoder);
+    CHIP_ERROR WriteListInt8uAttribute(TLV::TLVReader & aReader);
     CHIP_ERROR ReadListOctetStringAttribute(AttributeValueEncoder & aEncoder);
+    CHIP_ERROR WriteListOctetStringAttribute(TLV::TLVReader & aReader);
     CHIP_ERROR ReadListStructOctetStringAttribute(AttributeValueEncoder & aEncoder);
+    CHIP_ERROR WriteListStructOctetStringAttribute(TLV::TLVReader & aReader);
     CHIP_ERROR ReadListNullablesAndOptionalsStructAttribute(AttributeValueEncoder & aEncoder);
+    CHIP_ERROR WrtieListNullablesAndOptionalsStructAttribute(TLV::TLVReader & aReader);
 };
 
 TestAttrAccess gAttrAccess;
+uint8_t listUint8Data[kAttributeCount];
+char listOctetStringData[kAttributeCount][6];
```

Comment:
These are member variables - please prefix them with `m`.

Reply:
They are not member variables, but local shared global variables